src/components/crossDevice/CrossDeviceLink/style.css

```diff
@@ -1,42 +1,128 @@
-.linkSection{
-  margin-top: 48px;
-  text-align: left;
+.header {
+  margin-top: -71px;
+}
+
+.requestError {
+  margin-bottom: -11px;
+}
+
+.smsSection {
+  height: 76px;
+  width: 432px;
```

Comment:
Please could you remove these hard coded heights and widths? I think sections like this should fill the width of the container and take the height of all the objects in them.

Otherwise it makes responsive web pages much harder: if you change the right/left margin of the `onfido-sdk-ui-Theme-step` container everything inside it stays centered and resizes apart from the mobile input

Reply:
hard coded height in this case is fine. The width is also not a blocker IMO. Quite the edge case to go smaller than that width.
Responsiveness is not always straightforward to do.

Reply:
well, I take it back. This case is actually simple to use percentages and neater.

Reply:
and yet again, keeping the send button on the right with static width is added complexity.

Reply:
There is a ticket for visual improvements of the sms screen, you can add the issue there and I can handle it as part of that story.
https://jira.onfido.co.uk/browse/CX-1435